### ISSUE TITLE
Refined Storage Controller loses power even though its buffer if filled to the max.

### ISSUE THREAD

**Issue by Furious1964** [FTBTeam/FTB-Modpack-Issues#2891]
### Modpack

FTB StoneBlock 3

### Modpack version

1.6.1

### Describe the bug

Adding a new Wireless Transmitter to the Network causes the entire Network to flicker as if it's running out of power. Feeding Controller MAX power from my Flux Network.

### Steps to reproduce

1. Built a large Refined Storage Network over multiple Dimensions and numerous Crafters.
2. Add another Wireless Transmitter to the Network.
3. Watch as everything flickers as if there's not enough power.

### Expected behaviour

I expected it to work.

### Screenshots

![2023-04-01_14 01 42](https://user-images.githubusercontent.com/2803725/229307307-90c1c87e-eff5-4527-944c-b390ad33927b.png)
![2023-04-01_14 05 46](https://user-images.githubusercontent.com/2803725/229307309-91783d54-683c-4ed8-a92f-ba331dc02d7e.png)

### Additional information

Removing the additional Transmitter makes the Network work again.

**Comment by UnRealDinnerbone** [comment]
Even though you're outputting "MAX" power, are you sure that you're outputting enough power? You can see how much power RS needs by opening the controller. And you can see how much power the flux point it outputting by opening it also. The Wireless Transmitter uses quite a bit of power, so it might be pushing the point over the limit it is set to

**Comment by Furious1964** [comment]
I checked the power requirements for the Controller and it had a buffer size of 32K RF. The limit of the Point was set to 800K and I even disabled the limitation in order to give it all the power it needs. My power generation of close to 180M RF/Tick. I did look up other solution and the one that worked was to change the controllers buffer size to a ridiculous amount....I changed it to 99M and the problem disappeared.
